oc_cdtapi/BitbucketAPI/models/pull_request_delete.py

```
@@ -0,0 +1,87 @@
+    Do not edit the class manually.
+"""  # noqa: E501
+
+
+from __future__ import annotations
+import pprint
+import re  # noqa: F401
+import json
+
+from pydantic import BaseModel, ConfigDict, StrictInt
+from typing import Any, ClassVar, Dict, List, Optional
+from typing import Optional, Set
```

Ruff (0.15.1)

[error] Redefinition of unused `Optional` from line 21: `Optional` redefined here

Remove definition: `Optional`

(F811)